integration-tests/tests/run-catalog-workflow.js

```diff
@@ -20,6 +22,21 @@ const testCatalogFlowFn = _.flow(
   await dismissNotifications(page)
 
   await click(page, clickable({ textContains: 'browse & explore' }))
+  await waitForNoSpinners(page)
+  await click(page, clickable({ textContains: 'Granted' }))
+  await clickTableCell(page, "dataset list", 2, 2)
+  await waitForNoSpinners(page)
+  await click(page, clickable({ textContains: 'Link to a workspace' }))
+  await waitForNoSpinners(page)
+
+  await click(page, clickable({ textContains: 'Start with a new workspace' }))
+  await findText(page, 'Create a New Workspace')
+  await click(page, clickable({ textContains: 'Cancel' }))
+
+  await click(page, clickable({ textContains: 'Start with an existing workspace' }))
+  await click(page, input({ labelContains: 'Select a workspace' }))
+  await click(page, `//*[@role="combobox"][contains(normalize-space(.), "${workspaceName}")]`)
+  await click(page, clickable({ textContains: 'Import' }))
```

Comment:
To close out the journey, could we add two more items:
1. A few more steps to ensure we made it to the workspace page
    1. Something like another `waitForNoSpinners` and a check to ensure we made it to the workspace page without an error
2. Wrap it in a `withScreenshot` - this will take a screenshot in case an error is encountered.

Reply:
It looks like every test calls `withScreenshot` here: https://github.com/DataBiosphere/terra-ui/blob/dev/integration-tests/src/server.js#L50